lib/taurus/core/util/log.py

```diff
@@ -793,6 +793,15 @@ def _format_stack(self, stack_func=inspect.stack):
                         out += '<could not find suitable string representation>'
         return out
 
+    def loggingtofile(self, msg, logging_path, *args, **kw):
```

Comment:
IMHO this method's name should be written with `snake_case` and begin with a verb. What do you think of: `log_to_file`?